core/src/main/java/tech/pegasys/eth2signer/core/Runner.java

```diff
@@ -56,27 +59,43 @@ public Runner(final Config config) {
   public void run() {
 
     final Vertx vertx = Vertx.vertx();
+    final MetricsEndpoint metricsEndpoint =
+        new MetricsEndpoint(
+            config.isMetricsEnabled(),
+            config.getMetricsPort(),
+            config.getMetricsNetworkInterface(),
+            config.getMetricCategories(),
+            vertx);
     try {
-      final Handler<HttpServerRequest> requestHandler = createRouter(vertx);
+      final Handler<HttpServerRequest> requestHandler =
+          createRouter(vertx, metricsEndpoint.getMetricsSystem());
       final HttpServer httpServer = createServerAndWait(vertx, requestHandler);
       LOG.info("Server is up, and listening on {}", httpServer.actualPort());
 
+      metricsEndpoint.start();
```

Comment:
nit: is there a reason you star the metrics endpoint _after_ the server is started? I'd have thought we'd start metrics first ... 🤷‍♂

Reply:
no particular reason. don't think the order matters at all. is there a good reason to start it first?